userspace/libsinsp/event.h

```diff
@@ -31,11 +31,11 @@ typedef class sinsp_threadinfo sinsp_threadinfo;
 ///////////////////////////////////////////////////////////////////////////////
 typedef enum filtercheck_field_flags
 {
-	EPF_NONE = 0,
```

Comment:
When are they used as bitmask? I may be missing something, but these fields seem exclusives to me from their meanings.